src/main/java/io/github/Alathra/AlathranCommands/utils/PlaytimeChecker.java

```diff
@@ -1,8 +1,11 @@
 package io.github.Alathra.AlathranCommands.utils;
```

Comment:
I'm not objecting to putting this in `utils`, although due to the limited amount of code in this class I think it would be cleaner to put these methods in the `CommandPlaytime.java` class.

Up to you really.

Reply:
This class wasn't created for this command specifically. It also contains the method to check playtime for WildTP, which I didn't want in the class for WildTP - I felt it worked better as a separate class. Since it already existed I also felt I'd rather have the playtime for /playtime here as well.